middleware/localNamingConventions.js

```diff
@@ -1,5 +1,20 @@
+var check = require('check-types');
+
+var flipNumberAndStreetCountries = ['DEU', 'FIN', 'SWE', 'NOR', 'DNK', 'ISL'];
 
 function setup() {
+  var api = require('pelias-config').generate().api;
+  var settings = api.localization;
+  if (settings && settings.flipNumberAndStreetCountries) {
```

Comment:
I think the entire contents of this if statement can be made much shorter with lodash. put `var _ = require( 'lodash' );` at the top and then the body can become

```
var countries = settings.flipNumberAndStreetCountries;
flipNumberAndStreetCountries = _.uniq(flipNumberAndStreetCountries.concat(countries));
```